pages/permission_management.rst

```diff
@@ -0,0 +1,319 @@
+
+.. image:: /images/permissions/permissions_09.png
+
+The information which specific entity a user or team has access to is managed through “sharing” on the entity itself, not through a role.
+
+As an example, in 3.3 and earlier, to give access to a stream containing windows logs and the corresponding dashboard visualizing them, an administrator had to create a role:
+“Windows Logs”, having “Stream Windows Logs” as “Allow Reading”, and “Dashboard Windows Logs” as “Allow Reading”. This role was then assigned to a user, either manually or via a group mapping.
+
+In 4.0, there is no special role necessary for this access. Instead, the Administrator grants access to the stream, and either the Administrator or another owner of the dashboard shares access to the entities with a specific user or team. For most of the process, the user sharing the access does not have to have administrator-level access.
+
+Roles now only govern what actions someone can take, but do not themselves state on which entities these actions can take place. The latter is done through the sharing dialog. (see the later section for details)
+
+In 4.0 the UI does not allow defining new roles, even though this is still possible through the API. As there is much less need to create custom roles, we believe this is acceptable initially, but we plan on making custom roles possible in future releases.
+
+Providing Dashboard Creation Access
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Before users can create their own Dashboards, you need to provide them the appropriate level of access. 
+
+Under the “System” dropdown menu located in the top menu, click on the “Users and Teams” option. Choose the User record that you want to update. 
+
+.. image:: /images/permissions/permissions_10.png
+
+In the “Assign Roles” menu, you can change the individual user’s permissions to better align with their job function. In this case, the user, Alice, needs to be able to create Dashboards. Click on “Dashboard Creator,” then click “Assign Role.” Graylog automatically updates the user’s account, granting the necessary access immediately. 
+
+.. image:: /images/permissions/permissions_11.png
+
+After providing “Dashboard Creator” access to users, they will be able to see the “Create a Dashboard” button on the upper right-hand side of their Dashboards view.
+
+.. image:: /images/permissions/permissions_12.png
+
+**Example:** Manually Granting Access Permissions
+
+“Alice” is on the Security Team. However, before being added to a Team in Graylog, she has no access to Streams or Dashboards. 
+
+Alice’s User View before providing her access looks like this:
+
+.. image:: /images/permissions/permissions_13.png
+
+As a member of the Security Team, Alice needs access to email logs. As an Admin, you can go into Graylog, select the Security Team, and then select the appropriate level of access.
+
+Teams
+-----
+
+.. image:: /images/permissions/permissions_14.png
```

Comment:
This screenshot should be used instead of `.. image:: /images/permissions/permissions_16.png` below and removed from this position.
It merely shows the teams overview page and there is no need for it here.